bookdown/02-basics.Rmd

```diff
@@ -663,17 +660,18 @@ local({ # we do this locally to not overwrite the objects from previous chunks
 
 ### Performance assessment {#measure}
 
-The last step of an modeling is usually the performance assessment.
-The quality of the predictions of a model in `mlr3` can be assessed with respect to a number of different performance measures.
-At the performance assessment we choose a specific performance measure to quantify the predictions.
-This is done by comparing the predicted labels with the true labels.
```

Comment:
Here I had to rewrite most parts because it just sounded weird. Hopefully the new wording makes more sense.